### PR TITLE
feat(New Features) : Revamp PA API code snippets to support new features

### PR DESCRIPTION
**Pull request by vamshimupparaju**
As per new features in PA end points added those new features to code snippets.

### CODE REVIEW

**Review by AkankshaAcharya — APPROVED** [review]

**Review by ksreeramoj — APPROVED** [review]

**Review by Ramanth — APPROVED** [review]

File: examples/PAV3MultipleCalculation.R [hunk 1]
```diff
@@ -209,9 +244,10 @@ main <- function() {
     paste0(host,
            "/analytics/engines/pa/v3/components?document=",
            paDocument)
-  
+  paramlst <- list(document = paDocument)
```
Comment (saigiridhar21) [inline examples/PAV3MultipleCalculation.R]: Very minor: Can you please rename the variable to `params` or `queryParams` or something similar? It is always preferred to hide the datatypes in variable names.

Comment (saigiridhar21) [reply examples/PAV3MultipleCalculation.R]: Were you adding `document` query parameter at two places? I think that you can remove it from `componentsLookupUrl`.

File: examples/PAV3MultipleCalculation.R [hunk 2]
```diff
@@ -241,12 +277,61 @@ main <- function() {
...
+                   username,
+                   password,
+                   queryParams =
+                     paramlst)
+  
+  pricingSourcesResponseObj <-
+    jsonlite::fromJSON(httr::content(pricingSourcesResponse, "text", encoding = "UTF-8"))
+  pricingSourcesResponseData <- pricingSourcesResponseObj$data
+  
+  lstnames <- names(pricingSourcesResponseData)
```
Comment (saigiridhar21) [inline examples/PAV3MultipleCalculation.R]: Can you please rename the variable to `pricingSourceNames` or something similar?

File: examples/PAV3MultipleCalculation.R (hunk 1 above)
Comment (vamshimupparaju) [reply examples/PAV3MultipleCalculation.R]: for retrieving component Id passing paDocument only at this one place

**Review by saigiridhar21 — APPROVED** [review]

**Pull request merged**